graphics/cairomm/Portfile

```
@@ -1,35 +1,41 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+
 PortGroup           active_variants 1.1
+PortGroup           boost 1.0
+PortGroup           meson 1.0
 
 name                cairomm
-version             1.12.2
+version             1.16.2
 categories          graphics
 license             LGPL-2+
 maintainers         {devans @dbevans} openmaintainer
+platforms           darwin
+
 description         Cairo is a vector graphics library with cross-device output support.
 long_description    Cairo is designed to produce identical \
                     output on all output media while taking advantage of \
                     display hardware acceleration when available \
                     (e.g. through the X Render Extension).
+
 homepage            https://www.cairographics.org
 master_sites        ${homepage}/releases/
-platforms           darwin
 
-checksums           rmd160  112e58d903f168232b6a2673802b83ce687eb46c \
-                    sha256  45c47fd4d0aa77464a75cdca011143fea3ef795c4753f6e860057da5fb8bd599 \
-                    size    1325223
+checksums           rmd160  a3d749af53be09c5fbe629a394a69a70cfcf8359 \
+                    sha256  6a63bf98a97dda2b0f55e34d1b5f3fb909ef8b70f9b8d382cb1ff3978e7dc13f \
+                    size    593504
+
+use_xz              yes
 
 depends_build       port:pkgconfig \
                     port:mm-common
 
 depends_lib         path:lib/pkgconfig/cairo.pc:cairo \
+                    port:doxygen \
+                    port:graphviz \
                     port:libsigcxx2
```

doxygen would be a build dependency, not a lib dependency. doxygen sometimes uses graphviz, so graphviz would be a build dependency too. When you depend on graphviz, make sure to do it in a way that allows graphviz-devel to satisfy it; usually that's `path:bin/dot:graphviz`.

However doxygen and graphviz are only being called for because the port is trying to build its own copy of libsigc++ 3 because it didn't find one already installed. Switching the libsigcxx2 dependency to libsigcxx3 works for me:

```suggestion
                    port:libsigcxx3
```

---

Thank you! Done.